query-planner-js/src/composedSchema/toAPISchema.ts

```diff
@@ -14,27 +14,31 @@ declare module 'graphql' {
   }
 }
 
-export function toAPISchema(schema: GraphQLSchema): GraphQLSchema {
+export function toAPISchema(schema: GraphQLSchema): {
```

Comment:
This wasn't introduced by this PR, but I don't understand why this function is in `@apollo/query-planner`.  Shouldn't it be in `@apollo/federation`?  Seems odd for an implementor to need the query planner for applying such a transformation.

Reply:
It's part of `@apollo/query-planner` because it was extracted out of `buildComposedSchema`. I agree it doesn't really belong there, but I don't think it belongs in `@apollo/federation` either. The idea was that this was temporary hacky code and we'd move it to a new `@apollo/core` library soon. (We should also move composition out of `@apollo/federation`, to cleanly separate federated subgraph authoring from supergraph composition.)

Reply:
Ah, `@apollo/core` does seem like a reasonable consideration.  Happy to leave it here for now.